content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -499,7 +499,7 @@ spec:
     webhook:
       # conversionReviewVersions indicates what ConversionReview versions are understood/preferred by the webhook.
       # The first version in the list understood by the API server is sent to the webhook.
-      # The webhook must respond with a ConversionReview object in the same version it received.
+      # The webhook must respond with a ConversionResponse object in the same version it received.
```

Comment:
I'm not sure about this one. The `ConversionReview` contains the [request and response](https://github.com/kubernetes/apiextensions-apiserver/blob/1d1fb1b9cf4fa3a590e8d23264acb70b7786d38c/pkg/apis/apiextensions/v1/types.go#L476). Can you point out where just the response is returned?

Reply:
Ohh you are right! Thanks for catching that. I will fix it.

Reply:
Updated it.